lib/src/client.dart

```diff
@@ -2190,6 +2195,7 @@ class Client extends MatrixApi {
           await _handleRoomEvents(room, state, EventUpdateType.inviteState);
         }
       }
+      await database?.storeRoomUpdate(id, syncRoomUpdate, room.lastEvent, this);
     }
   }
 
```

Comment:
There isn't really a place to put it, but can you update the comment in the roomPreviewLastEvents.addAll call in this file and possibly resolve it, now that we don't set the state key to empty for those events anymore?

Reply:
there is no comment in the roomPreviewLastEvents.addAll call

Reply:
I meant this one: https://github.com/famedly/matrix-dart-sdk/blob/cf4dac79693a55883591d5e968c69473d0b7f6ea/lib/src/client.dart#L237

Reply:
Talked with @td-famedly about it and commented it in and removed the hack comment